### PR TITLE
Added Library Deletion Functionality

### PR DESCRIPTION
**Pull request by MattGill98**

### CODE REVIEW

File: src/main/java/fish/payara/sample/debugging/model/ServiceModel.java [hunk 1]
```diff
@@ -21,19 +24,26 @@ public ServiceModel() {
...
         return libraries;
     }
 
-    public Book addBook(Library library, Book book) {
-        library.addBook(book);
-        return addBook(book);
-    }
-
+    /**
+     * Adds a book to the store.
```
Comment (jake-walker) [inline src/main/java/fish/payara/sample/debugging/model/ServiceModel.java]:
```suggestion
     * Adds a book to the library.
```

Comment (jake-walker) [inline src/main/java/fish/payara/sample/debugging/model/ServiceModel.java]: 'Store' might be confusing as all the code refers to a 'library'.
```suggestion
     * Adds a book to the library.
```

File: src/main/java/fish/payara/sample/debugging/service/LibraryEndpoints.java [hunk 2]
```diff
@@ -48,4 +48,11 @@ public Book addBookToLibrary(String name, Book book) {
         return book;
     }
 
+    @Override
+    public Library deleteLibrary(String name) {
+        Library library = getLibrary(name);
+        library.getBooks().remove(library);
```
Comment (jake-walker) [inline src/main/java/fish/payara/sample/debugging/service/LibraryEndpoints.java]: Here a library object is being removed from a list of books. It should be removing the library from the list of libraries.
```suggestion
        model.getLibraries().remove(library);
```

**Comment by MattGill98** [comment]
Nicely reviewed! Feel free to merge this, and use whatever merge strategy you want - we can discuss why you chose whatever you do at some point!

**Pull request closed without merging**